pkg/logqlmodel/stats/stats.proto

```diff
@@ -57,6 +57,10 @@ message Caches {
     (gogoproto.nullable) = false,
     (gogoproto.jsontag) = "labelResult"
   ];
+  Cache instantMetricResult = 8 [
```

Comment:
Nit: this should be called `instantMetricQueryResult` IMHO.

Reply:
I tried to be in consistent with other cases here.

```
  Cache statsResult = 4 [
    (gogoproto.nullable) = false,
    (gogoproto.jsontag) = "statsResult"
  ];
  Cache volumeResult = 5 [
    (gogoproto.nullable) = false,
    (gogoproto.jsontag) = "volumeResult"
  ];
  Cache seriesResult = 6 [
    (gogoproto.nullable) = false,
    (gogoproto.jsontag) = "seriesResult"
  ];
  Cache labelResult = 7 [
    (gogoproto.nullable) = false,
    (gogoproto.jsontag) = "labelResult"
  ];
```
None of them have `Query` to it. I would prefer to keep it this way for consistency, I can change if you really want to.